game/src/constants.js

```diff
@@ -2,9 +2,11 @@ const gravity = new BABYLON.Vector2(0, -9.8);
 const flightForce = 5; // The amount of force applied when the "fly" button is pressed
 
 const greetings = [
+    'Let\'s gooooo!!!!'
```

Comment:
Needs a comma on the end!

Reply:
oops copied and pasted from last line to top line without looking. i know better ;) fixing!